database/migrations.go

```diff
@@ -402,6 +402,14 @@ var migrations = []func(tx *sql.Tx) error{
 		_, err = tx.Exec(sql)
 		return err
 	},
+	func(tx *sql.Tx) (err error) {
```

Comment:
SQL migrations are ordered. New changes should go at the bottom of the list.

Reply:
Thank you for pointing out the relevant mistakes. I force-pushed the branch to reflect the changes you requested. I hope that's okay.